JFrog updates, v1 draft, DOC-3625 (#3056)
harness/developer-hub: docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md

---

> title: Jfrog Artifact and JFrog xray scanner reference

Reviewer: ```suggestion
title: JFrog Artifactory and JFrog Xray scanner reference
```

Reviewer: these are product names

description: Image scans with Jfrog xray
sidebar_position: 330
---

> You can ingest scan results from JFrog Artifact and Jfrog xray. The standard workflow is to create a CI Build or Security Tests stage to your pipeline, add a Security step, and then use `setting:value` pairs to configure the step as specified below.

Reviewer: ```suggestion
You can ingest scan results from JFrog Artifactory and JFrog Xray. The standard workflow is to create a CI Build or Security Tests stage in your pipeline, add a Security step, and then use `setting:value` pairs to configure the step as specified below.
```

## Before you begin

:::info important notes

- Harness STO supports `ingestionOnly` scans with Jfrog Xray. `orchestrationOnly` and `dataLoad` scans are not supported. 

> - You need to run the scan step with root access if you need to add trusted certificates to your scan images at runtime. 

Reviewer: ```suggestion
- If you need to add trusted certificates to your scan images at runtime, then you need to run the scan step with root access. 
```

Reviewer: dependent clause goes 2nd
